BuildResidentialScheduleFile/resources/weekday/duration_probability/cooking_evening_duration_probability.csv

```diff
@@ -0,0 +1,37 @@
+0,2,0.26383140135425953
```

Comment:
This is it. Simply combined all the cluster_x_foo.csv into a single foo.csv where the first column is the cluster number.